### DISCUSSION TITLE
How to make  the goodness of fit plot

### DISCUSSION THREAD

**Discussion by Wenjing1018** [Open-Systems-Pharmacology/Forum#833] [Questions & Problems] [answered]
Deear all,
I plan to make  the goodness of fit plot,so I wonder if I should use individual or population models?
![image](https://user-images.githubusercontent.com/63133525/91730476-7d8bd200-ebd8-11ea-9959-e243302ba757.png)

**Comment by prvmalik** [comment, marked as answer]
At present, goodness of fit plots can only be made with individual simulations using the parameter identification tool.

However, you can export the results of population simulations to .csv files and make the graphs on your preferred choice of graphing software without any problems.

**Comment by AndreDlm** [comment]
There's another workaround that has been suggested some time ago by @PavelBal (I can't find the issue entry anymore):

Just set up a parameter identification with your simulation(s) and all observed data sets, set the maximum number of iterations in the configuration window to zero and click on run. Then you can select GOF plots, residual plots etc...

![Screenshot 2020-09-14 130525](https://user-images.githubusercontent.com/26541603/93078665-2df2df00-f68b-11ea-916a-ba04498fcdf2.png)

**Comment by prvmalik** [comment]
^^ Yes this is an example of using individual simulations in the parameter identification tool to get a GOF plot. But this does not work with population simulations.
